microsoft-edge/webview2/how-to/set-preview-channel.md

```diff
@@ -43,27 +45,30 @@ There are several ways to make your WebView2 app use a specified preview channel
 
 These approaches are described below.
 
+### Browser executable folder
+
+The browser executable folder option allows you to specify a known folder containing runtime binaries. This folder can be the installed location of the WebView2 Runtime, a preview channel of Microsoft Edge, or it can be a folder containing fixed version binaries that you have deployed to the machine yourself.
+
+If setting the browser executable folder to a specific preview channel of Microsoft Edge, you will need to update the location when that preview channel updates to a newer version. This is because the location includes the version as part of its path. For this reason, we recommend this approach for local testing only.
```

Comment:
```suggestion
If you set the browser executable folder to a specific preview channel of Microsoft Edge, you will need to update the location when that preview channel updates to a newer version. This is because the location includes the version number as part of its path. Therefore, we recommend using this approach for local testing only.
```